### PR TITLE
udpate laravel deps

### PR DESCRIPTION
**Pull request by chrissm79**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "barryvdh/laravel-async-queue",
+    "name": "nuwave/laravel-async-queue",
```
Comment (barryvdh) [inline composer.json]: This doesn't need to change

File: composer.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "5.3.x",
-        "illuminate/console": "5.3.x",
+        "illuminate/support": "5.4.x",
+        "illuminate/console": "5.4.x",
```
Comment (barryvdh) [inline composer.json]: Can you allow bot?

File: composer.json (hunk 1 above)
Comment (chrissm79) [inline composer.json]: Whoops, I'm an idiot!

**Comment by chrissm79** [comment]
Updated, thanks!!

**Comment by barryvdh** [comment]
Sorry to be so picky, but can you do 5.3.x|5.4.x ? Laravel doesn't follow SemVer so don't want to allow 5.5 etc just yet.

**Comment by chrissm79** [comment]
no worries, done!

**Pull request merged**